Source/Core/Core/HW/DSP.cpp

```diff
@@ -212,6 +212,7 @@ void EnableInstantDMA()
 	CoreTiming::RemoveEvent(et_CompleteARAM);
 	CompleteARAM(0, 0);
 	instant_dma = true;
+	ERROR_LOG(DSPINTERFACE, "Enabling Instant ARAM DMA hack");
```

Comment:
It's logs. Logs are meant for us, not for end users. I'm 100% fine with "user confusing" logs. Let's keep this one as it is.